.github/workflows/register-test.yml

```diff
@@ -0,0 +1,15 @@
+name: End-to-end tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v1
```

Comment:
did you try v2?

Reply:
Sorry bhai. Tried v2 and passed without an issue.